src/onnx/onnx.cpp

```diff
@@ -1024,12 +1016,11 @@ struct onnx_parser
             auto&& bias_floats = attributes["bias"].floats();
             bias               = std::vector<float>(bias_floats.begin(), bias_floats.end());
         }
```

Comment:
The changes after this line is the old code, which should not be here. This might be due to you copied the file from my branch to here. I think you may need a merge instead of copying the file.

Reply:
fixed